about/policy/open-sgid/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Open SGID Policy and Terms of Use
+author:
+  display_name: AGRC Office
+  email: agrc@utah.gov
+tags:
+  - policy
+  - sgid
+categories: []
+date: 2020-05-04 10:01:27 -0500
+---
+
+The following guidelines apply when using the Open SGID database. 
+
+## Do
+{: .text-left }
+
+- Search our [known-issues](https://github.com/agrc/open-sgid/issues) page if you are having trouble 
+- {% include contact.html subject=page.title text="Contact " contact=site.data.contacts.opensgid %} if you encounter a new issue.
+- Use for individual purposes only. Examples include:
```

Comment:
**suggestion**: what do you think about adding to the list or adjusting the some what dry language with things like make cool and fun maps, solve problems, automated repetitive tasks?

Reply:
yes!  let's not lose sight of the fact that our website lists one of our skills as [cartwheels](https://gis.utah.gov/images/Cartwheel.jpg).  i like what you're throwing down!

Reply:
👍  who would have thought guidelines could be exciting!? 👍

Reply:
notes to self (quick brainstorm for comically, yet serious uses to sift through and refine tomorrow).
  
Example uses include: 
- exploring the nation's 'greatest' (albeit, self-proclaimed) statewide database
- exploring a surprising diverse statewide database
- making cool maps
- finding answers to life's persistent questions
- producing things to inflate your resume
- impressing your mother/friends (total matt peters, there)
- exploring the human superpower of accessible data
- writing pompous spatial queries
- getting hip with open source

Reply:
I also keep getting hung up on the _individual purposes only_ part, and don't want it to be conflated with _personal use_, which could be interpreted as "excludes use for the benefit of any third party, including commercial, educational, governmental or non-profit entities".  Are we trying to reiterate not using live database connections in applications? (see @steveoh's comment above) 

What about something like:

- Use for a wide variety of individual GIS projects, such as:

Or just drop the word _individual_ altogether?

Reply:
good feedback - thanks nathan, steve!  I'll incorporate these comments into the next commit.    I'm going to mark this as resolved and we can start fresh from there (as this thread will be outdated).